types/react-helmet-async.d.ts

```diff
@@ -0,0 +1,22 @@
+declare module 'react-helmet-async' {
+  import * as React from 'react';
+
+  // @ts-ignore
+  import { Helmet, HelmetData } from 'react-helmet-async';
+
+  // @ts-ignore
+  export { Helmet, HelmetData };
+
+  // tslint:disable-next-line:interface-name
+  export interface PopulatedContext {
+    helmet: HelmetData;
+  }
+
+  // tslint:disable-next-line:interface-name
+  interface ProviderProps {
+    context?: {};
+  }
+
+  // @ts-ignore
+  export class HelmetProvider extends React.Component<ProviderProps> {}
+}
```

Comment:
This was needed because the typings were somehow out of sync, and was giving me
```
TS2604: JSX element type 'Helmet' does not have any construct or call signatures.
```